modal/mount.py

```diff
@@ -810,3 +810,110 @@ def _is_modal_path(remote_path: PurePosixPath):
         if is_modal_path:
             return True
     return False
+
+REMOTE_PACKAGES_PATH = "/__modal/deps"
+REMOTE_SITECUSTOMIZE_PATH = "/pkg/sitecustomize.py"
+
+SITECUSTOMIZE_CONTENT = f"""
+# This file is automatically generated by Modal.
+# It ensures that Modal's python dependencies are available in the Python PATH,
+# while prioritizing user-installed packages.
+import sys; sys.path.append('{REMOTE_PACKAGES_PATH}')
+""".strip()
+
+async def _create_single_mount(
+    client: _Client,
+    builder_version: str,
+    python_version: str,
+    platform: str,
+    arch: str,
+    uv_python_platform: str = None,
+    check_if_exists: bool = True,
+):
+    import subprocess
+    import tempfile
+
+    profile_environment = config.get("environment")
+
+    abi_tag = "cp" + python_version.replace(".", "")
+    mount_name = f"{builder_version}-{abi_tag}-{platform}-{arch}"
+    uv_python_platform = uv_python_platform or f"{arch}-{platform}"
+
+    if check_if_exists:
```

Comment:
I'm not sure why, but in the integration test setup this `from_name` seems to block the event loop and everything croaks. It's strictly not needed in there, so passing this flag was a quick fix to get around it.